docs/docs/references/configuration/cli/trivy_filesystem.md

```diff
@@ -59,6 +59,7 @@ trivy filesystem [flags] PATH
       --registry-token string       registry token
       --rekor-url string            [EXPERIMENTAL] address of rekor STL server (default "https://rekor.sigstore.dev")
       --reset                       remove all caches and database
+      --runtime strings             Runtime(s) to use (docker,containerd,podman,remote) (default [docker,containerd,podman,remote])
```

Comment:
we need this option only for `image` mode.
You can disable flag for other modes.
https://github.com/aquasecurity/trivy/blob/cc18f92cf39534a2220883722fb26dc1a9e27fda/pkg/commands/app.go#L299

Reply:
I moved this flag over to the `image` group, and regenerated the docs.

Reply:
It make sense. Good solution.